multipaz/src/commonMain/kotlin/org/multipaz/asn1/ASN1Integer.kt

```diff
@@ -42,6 +44,23 @@ class ASN1Integer(
         fun parse(content: ByteArray, tag: Int): ASN1Integer {
             return ASN1Integer(content, tag)
         }
+
+        /**
+         * Generates a [ASN1Integer] with [numBits] bits of random
```

Comment:
We're actually providing a little less than numBits, because we only use 7(-ish) bits of randomness in the first byte. Does it make more sense to use the full number of bits, or to add a sentence here explaining that we actually only use numBits-1?